lib/google_drive.rb

```diff
@@ -1,6 +1,10 @@
 # Author: Hiroshi Ichikawa <http://gimite.net/>
 # The license of this source is "New BSD Licence"
 
+Note that when using the OAuth flow, the client will be able to access all files belonging to the the user who generated the token. To set permissions more specifically, you may want to consider using a service account.
```

Comment:
Putting this line here would cause syntax error in Ruby. Maybe you can add it just above the comment you added below?